py3status/modules/uptime.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed
+
+@author Alexis "Horgix" Chotard <alexis.horgix.chotard@gmail.com>
+@license BSD
+"""
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    format = "{days} days, {hours} hours, {minutes} min"
+
+    def _uptime(self):
+        """Returns uptime in seconds or None, on Linux."""
+        with open('/proc/uptime', 'r') as f:
+            up = float(f.readline().split()[0])
+        return up
```

Comment:
Hi tobes,

I have the habit to keep things separated, but I guess that in this case it makes sense to avoid using another function. Merge in `uptime` main function.